README.md

```diff
@@ -38,4 +38,45 @@ flask run
 
 The server will start running at http://127.0.0.1:5000/ by default. 
 
-That's it! You should now have the Flask backend server up and running on your local machine.
\ No newline at end of file
+That's it! You should now have the Flask backend server up and running on your local machine.
+
+
+# Frontend Setup
```

Comment:
Frontend setup would be higher than backend setup :) I think it'll be easy to read